src/shims/runtime/logger.js

```diff
@@ -0,0 +1,22 @@
+'use strict'
+
+function getLogger () {
+  return {
+    debug: function (...args) {
+      console.debug(...args)
+    },
+    info: function (...args) {
+      console.info(...args)
+    },
+    warn: function (...args) {
+      console.warn(...args)
+    },
+    error: function (...args) {
+      console.error(...args);
+    }
+  };
+}
+
+module.exports = {
+  getLogger
+}
```

Comment:
(commenting here to get a thread)

Looks like a surgical change, thanks for using this approach 🚀

The CI is broken, here are the commands we run:
https://github.com/testground/sdk-js/blob/master/.github/workflows/main.yml

Reply:
(if you have the bandwidth, one follow-up PR / Task, could be to add a testground test to this repo, similar to what we do in the rust-sdk repo)

Reply:
Created an issue for it: https://github.com/testground/sdk-js/issues/27

Will do it once the browser-node plan example is in a "finished" state or whenever I find extra bandwidth sooner.